lib/matplotlib/widgets.py

```diff
@@ -1823,9 +1825,11 @@ def press(self, event):
             self._prev_event = event
             key = event.key or ''
             key = key.replace('ctrl', 'control')
-            # move state is locked in on a button press
-            if key == self.state_modifier_keys['move']:
-                self.state.add('move')
+            # move/rotate state is locked in on a button press
+            for action_key in ['move', 'rotate']:
```

Comment:
Shouldn't this rather be named
```suggestion
            for action in ['move', 'rotate']:
```
?

Reply:
Yes, this is definitely better to avoid confusion between dictionary and keyboard keys 😄 - changed.